content/07.pathogenesis.md

```diff
@@ -582,6 +582,67 @@ Data and empirically determined biological mechanisms inform models, while model
 Many current efforts to model R~t~ have also led to tools that assist the visualization of estimates in real time or over recent intervals [@url:https://epiforecasts.io/covid; @url:https://rt.live].
 These are valuable resources, yet it is also important to note that the estimates arise from models containing many assumptions and are dependent on the quality of the data they use, which varies widely by region.
 
+### Effect of Vaccines on Pathogenesis and Community Spread
+
+The vaccine clinical trial data demonstrate a significant reduction in the likelihood of contracting symptomatic COVID-19, thereby succeeding in the primary goal of vaccination.
+The mRNA vaccines in particular were initially so effective in preventing disease that they were also assumed to have an effect on the likelihood of transmission (e.g., venues requiring proof of vaccination).
+However, in light of the reduced efficacy in response to VOC, it is especially important to consider whether this assumption is supported by the available evidence.
+
+This question is made up of several components.
+The crux is whether vaccinated individuals with a SARS-CoV-2 infection, regardless of symptom status, are as contagious as unvaccinated, infected individuals.
+Additionally, as outlined above, an important qualification is that the variants of SARS-CoV-2 circulating at the time of each study must be considered in light of the effect of evolution on vaccine efficacy.
+
+The phase II/III clinical trials evaluating the mRNA vaccines assessed vaccine efficacy based on COVID-19 diagnosis, thereby detecting only patients who received a diagnosis.
+In order to identify patients infected with SARS-CoV-2 who did not receive a diagnosis, for example, potentially those who did not develop symptoms, it would be necessary to conduct routine PCR testing even in the absence of symptoms. <!-- Define PCR? -->
```

Comment:
Probably unnecessary to define here? But it might be useful to discuss further down where you mention cycle thresholds and interpreting tests.